optimize/util/optimize-commons/src/main/java/io/camunda/optimize/service/util/configuration/ConfigurationValidator.java

```diff
@@ -7,9 +7,30 @@
  */
 package io.camunda.optimize.service.util.configuration;
 
+import io.camunda.optimize.service.exceptions.OptimizeConfigurationException;
+import java.util.regex.Pattern;
+
 public class ConfigurationValidator {
 
+  private static final Pattern INVALID_INDEX_PREFIX_CHARS = Pattern.compile("[\\\\/*?\"<>| _]");
+
   public void validate(final ConfigurationService configurationService) {
     configurationService.getEmailAuthenticationConfiguration().validate();
+
+    validateIndexPrefix(configurationService.getElasticSearchConfiguration().getIndexPrefix());
+    validateIndexPrefix(configurationService.getOpenSearchConfiguration().getIndexPrefix());
+  }
+
+  private void validateIndexPrefix(final String indexPrefix) {
+    if (indexPrefix != null) {
```

Comment:
[very nit]

The option:
```
if (indexPrefix == null) {
  return;
}
```
would save an indentation level for the rest of the method